.github/ISSUE_TEMPLATE/new-documentation-page-suggestion.md

```diff
@@ -0,0 +1,16 @@
+---
+name: New Documentation Page Suggestion
+about: Suggest a new documentation page
+title: 'New Page: PAGE DESCRIPTION'
+labels: new_docs
+assignees: ''
+
+---
+
+**Briefly describe the contents of the page**
+
+**Which type of documentation is this?**
+<!-- Choose from: tutorial, recipe, concept, reference -->
+
+**Is there prior art? If so, list any relevant materials**
```

Comment:
Prior art? Is this referring to images and content to embed? Maybe it would be better stated as simply relevant information.

** If any relevant information already exists, please list below**

Reply:
Prior art is the term for "has this or something like it been done before". So this heading is asking, for example, if you're suggesting a new tutorial to link other tutorials on the same subject. "Prior art" isn't actually about media, but I suppose we could use a clearer heading.

Reply:
Actually I forgot that there's a comment right below this heading clarifying what we're asking for.

Reply:
While the sentence below aids in clearing the confusion, a differently formed sentence may reduce the need to clarify from the start. This may also reduce issue across language translation errors that could occur for the reader.
It could perhaps be written like so:
Has this idea been previously documented? If so, please provide a list of any relevant materials.